xilem/src/property_tuple.rs

```diff
@@ -154,7 +82,45 @@ macro_rules! __declare_property_tuple_loop {
             $Type: ident, $idx: tt;
         )+
     ) => {
-        type $Props = ($(Option<$Type>,)+);
+
+        #[expect(missing_docs)]
+        pub struct $Props($(Option<$Type>,)+);
```

Comment:
There are reasonable arguments for making this a struct with named fields. But I don't think that's especially worthwhile